cardano-cli/test/cardano-cli-golden/Test/Golden/Babbage/Governance/VerifyPoll.hs

```diff
@@ -19,15 +19,15 @@ import qualified Hedgehog.Internal.Property as H
 
 {- HLINT ignore "Use camelCase" -}
 
-hprop_golden_shelleyGovernanceVerifyPoll :: Property
-hprop_golden_shelleyGovernanceVerifyPoll = propertyOnce $ do
-  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/polls/basic.json"
-  txFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/verify/valid"
+hprop_golden_babbageGovernanceVerifyPoll :: Property
+hprop_golden_babbageGovernanceVerifyPoll = propertyOnce $ do
+  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/babbage/governance/polls/basic.json"
+  txFile <- noteInputFile "test/cardano-cli-golden/files/golden/babbage/governance/verify/valid"
```

Comment:
Don't include `babbage` as a top-level directory.

If you must include the era in the name because you are testing multiple eras (which isn't the case here), put the era at the lowest level in the filename instead.

This might see counter-intuitive because our commands have era at the top-level, but actually the era gets passed to the command and argument so operationally, it's more like run this command, and here is the era the command should run in.

Moreover, it is often the case commands behave the same in multiple eras so it isn't necessary to qualify by era.

Reply:
Thanks for the explanations :pray: I removed the `babbage` segment from paths.